slices/libpython3.8-stdlib.yaml

```diff
@@ -0,0 +1,381 @@
+      /usr/lib/python3.8/lib-dynload/audioop.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/lib-dynload/ossaudiodev.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/sndhdr.py:
+      /usr/lib/python3.8/sunau.py:
+      /usr/lib/python3.8/wave.py:
+
+  # Internet Data Handling
+  # https://docs.python.org/3.8/library/netdata.html
+  net-data:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/binhex.py:
+      /usr/lib/python3.8/json/**:
+      /usr/lib/python3.8/lib-dynload/_json.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/mailbox.py:
+      /usr/lib/python3.8/mailcap.py:
+
+  # Numeric and Mathematical Modules
+  # https://docs.python.org/3.8/library/numeric.html
+  numeric:
+    essential:
+      - libpython3.8-stdlib_core
+      - libmpdec2_libs
+    contents:
+      /usr/lib/python3.8/_pydecimal.py:
+      /usr/lib/python3.8/decimal.py:
+      /usr/lib/python3.8/fractions.py:
+      /usr/lib/python3.8/lib-dynload/_decimal.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/numbers.py:
+      /usr/lib/python3.8/statistics.py:
+
+  # Shared OS X support functions
+  osx-support:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/_osx_support.py:
+
+  # Data Persistence
+  # https://docs.python.org/3.8/library/persistence.html
+  persistence:
+    essential:
+      - libpython3.8-stdlib_core
+      - libdb5.3_libs
+      - libsqlite3-0_libs
+    contents:
+      /usr/lib/python3.8/dbm/**:
+      /usr/lib/python3.8/lib-dynload/_dbm.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/lib-dynload/_sqlite3.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/shelve.py:
+      /usr/lib/python3.8/sqlite3/**:
+
+  # pydoc - Documentation generator and online help system
+  # https://docs.python.org/3.8/library/pydoc.html
+  pydoc:
+    essential:
+      - libpython3.8-stdlib_core
+    contents:
+      /usr/lib/python3.8/pydoc_data/**:
+      /usr/lib/python3.8/pydoc.py:
+
+  # Text Processing Services
+  # https://docs.python.org/3.8/library/text.html
+  text:
+    essential:
+      - libpython3.8-stdlib_core
+      - libreadline8_libs
+      - libtinfo6_libs
+    contents:
+      /usr/lib/python3.8/difflib.py:
+      /usr/lib/python3.8/lib-dynload/readline.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/rlcompleter.py:
+
+  # Unix Specific Services
+  # https://docs.python.org/3.8/library/unix.html
+  unix:
+    essential:
+      - libpython3.8-stdlib_core
+      - libpython3.8-stdlib_frameworks
+      - libcrypt1_libs
+    contents:
+      /usr/lib/python3.8/crypt.py:
+      /usr/lib/python3.8/lib-dynload/_crypt.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/lib-dynload/nis.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/lib-dynload/resource.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/lib-dynload/termios.cpython-38-*-linux-*.so:
+      /usr/lib/python3.8/pipes.py:
+      /usr/lib/python3.8/pty.py:
+      /usr/lib/python3.8/tty.py:
+
+  extras:
```

Comment:
can you add a quick comment with the rationale for having this extra slice composed of outliers and why they are outliers?

Reply:
Added in https://github.com/canonical/chisel-releases/pull/44/commits/59ea763199ebca5f334753d40b701d828b7ea99a.